development/build/scripts.js

```diff
@@ -1325,7 +1325,8 @@ function renderHtmlFile({
   const htmlFilePath = `./app/${htmlName}.html`;
   const htmlTemplate = readFileSync(htmlFilePath, 'utf8');
 
-  const htmlOutput = Sqrl.render(htmlTemplate, { isMMI, isTest });
+  const eta = new Eta();
+  const htmlOutput = eta.renderString(htmlTemplate, { isMMI, isTest });
```

Comment:
The interface is _almost_ as simple as `squirrelly`; i've asked if they'd add static helpers: https://github.com/eta-dev/eta/issues/275